Code/GraphMol/MolDraw2D/DrawText.cpp

```diff
@@ -538,11 +532,11 @@ void DrawText::getStringRects(const std::string &text, OrientType orient,
     alignString(TextAlignType::START, draw_modes, rects);
   } else {
     double running_y = 0;
-    for (size_t i = 0; i < text_bits.size(); ++i) {
+    for (auto tb : text_bits) {
```

Comment:
can be `const auto &tb`, right? No need to copy those strings
same thing for line 528 above, but I can't make a comment on that line